v4/Providers/MorphServiceProvider.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace v4\Providers;
+
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Support\ServiceProvider;
+
+class MorphServiceProvider extends ServiceProvider
+{
+    public function boot() {
+        Relation::morphMap([
```

Comment:
so by default polymorphic relations use the fully qualified names of the classes, but we want it to not depend on whatever namespace changes we make so we do a mapping here. It's also nicer to read 'survey' than v4\Models\Survey in responses really